CHANGELOG.md

```diff
@@ -7,6 +7,34 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Add support for `monitoring_config` block
+- Add support for `managed_prometheus` block inside `monitoring_config`
```

Comment:
this is a neat new feature!

Reply:
btw shall this be added to https://github.com/mineiros-io/terraform-google-gke-cluster also or is this just available for autopilot clusters?

Reply:
if it's available, please add a ticket to the backlog :)

Reply:
upon discussion with @mariux: would it make sense to merge these two modules with a refactor?

Reply:
back in the days we explicitly decided to not do that because a lot of the autopilot options conflict with the non-autopilot options, and the implementation got pretty messy in the first place. In the initial implementation in which we tried to implement a generic module, we ended up having tons of conditions for those conflicting arguments which ended up causing a messy implementation. 

So based on my own learnings: would **strongly** advice  against merging the modules

Reply:
would it worth to revise this based on the documentation or nothing changed that much since?

Reply:
I don't think much has changed, simply by looking at the provider ... a few new features have been added but the conflicts attribute wise still remain